src/testcafe/schema.d.ts

```diff
@@ -51,3 +52,12 @@ export interface Screenshots {
     pathPattern?: string;
     fullPage?: boolean;
 }
+
+interface CompilerOptions {
```

Comment:
This file references types from the `testcafe` module. So, you can use definitions from it and remove the `CompilerOptions` and `TypeScriptCompilerOptions` interfaces.